tuplex/python/src/PythonContext.cc

```diff
@@ -95,6 +102,9 @@ namespace tuplex {
             numBytesSerialized += sizeof(double);
         }
 
+        auto newExceptions = _badParallelizeObjects.size() - prevNumExceptions;
```

Comment:
this is a count, isn't it? Maybe name a bit better. newExceptions doesn't reallt say anything except that it maybe is a collection of exceptions which are new?